### DISCUSSION TITLE
Last.FM configuration exposes credentials

### DISCUSSION THREAD

**Discussion by jelinj8** [rern/rAudio#107] [Fixed Issues]
Hello,
sorry for such a load of reports, I've found another bug while experimenting with my Audio build.
I've set the Last.FM credentials, when revisiting the settings, the pwd is shown in cleartext in the username field:
![image](https://user-images.githubusercontent.com/3816495/104462473-e6388180-55b0-11eb-98a3-e4c5627c85aa.png)

**Comment by rern** [comment]
Fixed.
